packages/react-native-drawer-layout/src/views/modern/Drawer.tsx

```diff
@@ -337,7 +337,7 @@ export function Drawer({
               },
             ],
     };
-  });
+  }, [touchStartX, drawerWidth, gestureState, drawerPosition, translationX]);
```

Comment:
Here it's:
`[layout, drawerWidth, drawerType, translateX, drawerPosition, isRTL]`.